pkg/gadgets/trace/tcpdrop/types/types.go

```diff
@@ -20,45 +20,57 @@ import (
 )
 
 type Event struct {
-	// Can't use eventtypes.Event because this gadget doesn't support filtering by containers
+	eventtypes.Event
+	eventtypes.WithMountNsID
+	eventtypes.WithNetNsID
```

Comment:
Not related to this specific PR but something we need to fix -> https://github.com/inspektor-gadget/inspektor-gadget/issues/1548